### DISCUSSION TITLE
Is it possible to show components other than text in Select or ComboBox?

### DISCUSSION THREAD

**Discussion by hhhhzs** [UI5/webcomponents#6256] [Q&A]
Our UX provided a design like this, which requires to put some components such as image and label inside the selector and options, but when I tried to add some child components (like the code below) it seems that only text can be shown in Select and ComboBox. Is there any way to show components other than text in Select or ComboBox? Thanks!
<img width="146" alt="image" src="https://user-images.githubusercontent.com/57512939/211269520-8c2475e5-4c26-4813-99fc-9b515884ceb2.png">
<img width="324" alt="image" src="https://user-images.githubusercontent.com/57512939/211269313-e58aa1b5-79cd-4459-9cee-2dac650b4f7f.png">
``` 
 <ComboBox>
        <FlexBox>
        <img src={} />
        <Label>content</Label>
        </FlexBox>
   </ComboBox>

**Comment by Lukas742** [comment]
Hi @hhhhzs 

I'll forward this question to our UI5 Web Components Colleagues as the affected component is developed in their repository.
